torchao/csrc/cuda/sparse_marlin/mem.h

```diff
@@ -27,6 +27,16 @@ __device__ inline void cp_async4_pred_zfill(void* smem_ptr,
                                             const bool zfill = false) {
   const int BYTES = 16;
   int src_in_bytes = (zfill ? 0 : BYTES);
+  #ifdef USE_ROCM
+  uint32_t smem = static_cast<uint32_t>(__builtin_amdgcn_s_getpc());
```

Comment:
I guess we are not using smem_ptr here. we need to find a proper equivalent.

Reply:
thanks , I have a fix following your suggestion.